include/seqan3/range/container/concatenated_sequences.hpp

```diff
@@ -21,9 +21,9 @@
 #include <seqan3/range/container/concept.hpp>
 #include <seqan3/range/detail/random_access_iterator.hpp>
 #include <seqan3/range/views/as_const.hpp>
-#include <seqan3/range/views/join.hpp>
 #include <seqan3/range/views/repeat_n.hpp>
 #include <seqan3/range/views/slice.hpp>
+#include <seqan3/utility/views/join_with.hpp>
```

Comment:
Is this even used in this class?

Reply:
I was cold. Needed my computer to heat up ;-)
Yes, not needed at all, no clue why I added it.